.npmignore

```diff
@@ -0,0 +1,7 @@
+/assets
+/blueprints/.jshintrc
+/dev
+/docs
+/tests/!(helpers)
+/tmp
+/*.*
```

Comment:
all root level files, aside from the automatically excluded readme, changlog, license, and package.json